pkg/psmdb/backup/pbm.go

```diff
@@ -104,34 +141,31 @@ func (b *PBM) SetConfig(stg api.BackupStorageSpec, pitr api.PITRSpec) error {
 		if err != nil {
 			return errors.Wrap(err, "getting s3 credentials secret name")
 		}
-		conf := pbm.Config{
-			PITR: pbm.PITRConf{
-				Enabled: pitr.Enabled,
-			},
-			Storage: pbm.StorageConf{
-				Type: pbm.StorageS3,
-				S3: s3.Conf{
-					Region:      stg.S3.Region,
-					EndpointURL: stg.S3.EndpointURL,
-					Bucket:      stg.S3.Bucket,
-					Prefix:      stg.S3.Prefix,
-					Credentials: s3.Credentials{
-						AccessKeyID:     string(s3secret.Data[awsAccessKeySecretKey]),
-						SecretAccessKey: string(s3secret.Data[awsSecretAccessKeySecretKey]),
-					},
+		conf.Storage = pbm.StorageConf{
+			Type: pbm.StorageS3,
+			S3: s3.Conf{
+				Region:      stg.S3.Region,
+				EndpointURL: stg.S3.EndpointURL,
+				Bucket:      stg.S3.Bucket,
+				Prefix:      stg.S3.Prefix,
+				Credentials: s3.Credentials{
+					AccessKeyID:     string(s3secret.Data[awsAccessKeySecretKey]),
+					SecretAccessKey: string(s3secret.Data[awsSecretAccessKeySecretKey]),
 				},
 			},
 		}
-		err = b.C.SetConfig(conf)
-		if err != nil {
-			return errors.Wrap(err, "write config")
-		}
 	case api.BackupStorageFilesystem:
 		return errors.New("filesystem backup storage not supported yet, skipping storage name")
 	default:
 		return errors.New("unsupported backup storage type")
 	}
 
+	conf.Backup.Priority = priority
```

Comment:
you can set it on line 129 when you are init the config

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/750/commits/dc51cc347ec473eab8dcf600b9ea825a6abbfca3